clients/pull_request.go

```diff
@@ -36,5 +37,6 @@ type Label struct {
 
 // Review represents a PR review.
 type Review struct {
-	State string
+	Author User
```

Comment:
It looks like we are not using this field anywhere? If you plan to add the usage of this in a follow up PR remove this field for now to avoid dead code.

Reply:
it's used in the Code-Review check.

Reply:
I mis-understood. You're correct, I'll remove it.

Reply:
done.